src/mon/LogMonitor.h

```diff
@@ -206,6 +206,8 @@ class LogMonitor : public PaxosService,
       "mon_cluster_log_to_graylog",
       "mon_cluster_log_to_graylog_host",
       "mon_cluster_log_to_graylog_port",
+      "fsid",
```

Comment:
@tchaikov You are right. These two fields are not used and tracked. So we shall remove them from the `handle_conf_change` method.